src/bees-resolve.cc

```diff
@@ -172,7 +172,7 @@ BeesResolver::adjust_offset(const BeesFileRange &haystack, const BeesBlockData &
 
 		// Found the hash but not the data.  Yay!
 		m_found_hash = true;
-		BEESLOG("HASH COLLISION\n"
+		BEESLOGWARN("HASH COLLISION\n"
```

Comment:
This should probably be INFO.  Some file formats use a 4K page size and CRC64 internally, and a user who has such a file will get thousands of these.

Reply:
Fixed as INFO.